modules/react/collection/lib/useCursorListModel.tsx

```diff
@@ -315,12 +315,12 @@ export const useCursorListModel = createModelHook({
   },
   requiredConfig: useBaseListModel.requiredConfig,
 })(config => {
-  const [cursorId, setCursorId] = React.useState(config.initialCursorId);
-  const columnCount = config.columnCount || 0;
   const list = useBaseListModel(config);
-  const initialCurrentRef = React.useRef(
+  const [cursorId, setCursorId] = React.useState(
     config.initialCursorId || (config.items?.length ? getId(list.state.items![0]) : '')
   );
+  const columnCount = config.columnCount || 0;
+  const initialCurrentRef = React.useRef(cursorId);
```

Comment:
I'm not sure why the initial `cursorId` could only be `config.initialCursorId`, but the `initialCurrentRef` could be `config.initialCursorId` OR the first item in the items array. This change makes it so `cursorId` can also start as the first item in the array.

This fixes any component using the dynamic API (passing `items`) from the "multiple tab stop" problem where no `cursorId` is defined yet until the user interacts with the collection. You can see that further down the `registerItem` method calls `setCursorId` to make sure there's always a cursor, but the dynamic API doesn't call `registerItem`.